.github/minitest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "minitest",
+      "pattern": [
+        {
+          "regexp": "^.*?\\[(.*?):(\\d+)\\]:$",
+          "file": 1,
+          "line": 2
+        },
+        {
+          "regexp": "^\\s*((?:Expected|Actual).*)$",
+          "message": 1,
+          "loop": true
```

Comment:
Due to a limitation in the format of problem matchers, it's currently not possible to extract multiline messages. For input such as:
```
TestTestApp#test_resources_pod_returns_spec_path [test/test_test_app.rb:31]:
Expected: ".."
  Actual: "."
```
"Expected: …" and "Actual: …" are two separate errors with this matcher. We are currently not able to match output such as:
```
TestTestApp#test_resources_pod_writes_podspec [test/test_test_app.rb:50]:
--- expected
+++ actual
@@ -1 +1 @@
-["dist/assetd", "dist/main.jsbundle"]
+["dist/assets", "dist/main.jsbundle"]
```
This issue has been reported in https://github.com/actions/toolkit/issues/193 and https://github.com/actions/toolkit/issues/319.